core/spec.md

```diff
@@ -3189,6 +3191,50 @@ and the following Resource level attributes:
     be silently ignored. See [Registry APIs](#registry-apis) for more
     information.
 
+##### `compatibility` Attribute
+- Type: String (with a defined enumeration of possible values)
```

Comment:
Perhaps we should say "resource-specified enumeration of possible values" (or similar - resource may be the wrong word here)? Given that a specification can extend as well as contract the list below, *any* value could be included.

Reply:
went with "resource-specified enumeration of possible values"